= ex04 =========================================================================

$> /usr/bin/norminette -R CheckForbiddenSourceHeader ft_convert_base2.c ft_convert_base.c | grep -E '^(Error|Warning)'


$> gcc -Wextra -Wall -Werror ft_convert_base2.c ft_convert_base.c main.c -o user_exe 



_atoi_base
_get_digit
_is_valid
_putnbr_base
_putnbr_base.len
_putnbr_base.tmp
_putnbr_base.to



_ft_convert_base



malloc



atoi_base
is_valid
putnbr_base


= Test 1 ===================================================
total 40
-rwxr-xr-x  1 deepthought  deepthought  17048 Sep 11 13:25 m1kajo6ytocpomt1pbrhkjep


$> ./m1kajo6ytocpomt1pbrhkjep
652
052
052
-13
-10
1201
55
-2
22
240033
040033
647033
047033
0
0
1
10011010010
10011010010
(null)
(null)
(null)
(null)
(null)
(null)
(null)
(null)
(null)
(null)
(null)
(null)
1N011010010
(null)
.N011010010
/N011010010
/N011010010
9b011010010
J
4
/
(null)
,
.Vo
aE*
sE*
nE*
kE*
34*
N4*
.X
8X
:X
1w
,k
,:
979J
,79J
;8K;
18K;
A8K;
xCK;
.CK;
SCK;
mCK;
v;K;
,~K;
.~K;
1~K;
7
M
XNNXNXNNNNXN
,PNXNXNNNNXN
2
7&
e&
,&
7&
0&
.0
Cbb


$> diff -U 3 user_output_test1 test1.output | cat -e
--- user_output_test1    2022-09-11 13:25:59.000000000 +0000$
+++ test1.output    2022-09-11 13:25:59.000000000 +0000$
@@ -1,16 +1,16 @@$
652$
-052$
-052$
+0$
+0$
-13$
-10$
1201$
55$
-2$
-22$
+2$
240033$
-040033$
-647033$
-047033$
+0$
+647$
+0$
0$
0$
1$
@@ -28,12 +28,12 @@$
(null)$
(null)$
(null)$
-1N011010010$
+1N$
(null)$
-.N011010010$
-/N011010010$
-/N011010010$
-9b011010010$
+.$
+/$
+/$
+9b$
J$
4$
/$
@@ -41,39 +41,39 @@$
,$
.Vo$
aE*$
-sE*$
-nE*$
-kE*$
-34*$
-N4*$
+s$
+n$
+k$
+34$
+N$
.X$
-8X$
-:X$
+8$
+:$
1w$
,k$
,:$
979J$
-,79J$
+,$
;8K;$
-18K;$
-A8K;$
-xCK;$
-.CK;$
-SCK;$
-mCK;$
-v;K;$
-,~K;$
-.~K;$
-1~K;$
+1$
+A$
+xC$
+.$
+S$
+m$
+v;$
+,~$
+.$
+1$
7$
M$
XNNXNXNNNNXN$
-,PNXNXNNNNXN$
+,P$
2$
7&$
-e&$
-,&$
-7&$
-0&$
+e$
+,$
+7$
+0$
.0$
Cbb$


--- user_output_test1    2022-09-11 13:25:59.000000000 +0000$
+++ test1.output    2022-09-11 13:25:59.000000000 +0000$
@@ -1,16 +1,16 @@$
652$
-052$
-052$
+0$
+0$
-13$
-10$
1201$
55$
-2$
-22$
+2$
240033$
-040033$
-647033$
-047033$
+0$
+647$
+0$
0$
0$
1$
@@ -28,12 +28,12 @@$
(null)$
(null)$
(null)$
-1N011010010$
+1N$
(null)$
-.N011010010$
-/N011010010$
-/N011010010$
-9b011010010$
+.$
+/$
+/$
+9b$
J$
4$
/$
@@ -41,39 +41,39 @@$
,$
.Vo$
aE*$
-sE*$
-nE*$
-kE*$
-34*$
-N4*$
+s$
+n$
+k$
+34$
+N$
.X$
-8X$
-:X$
+8$
+:$
1w$
,k$
,:$
979J$
-,79J$
+,$
;8K;$
-18K;$
-A8K;$
-xCK;$
-.CK;$
-SCK;$
-mCK;$
-v;K;$
-,~K;$
-.~K;$
-1~K;$
+1$
+A$
+xC$
+.$
+S$
+m$
+v;$
+,~$
+.$
+1$
7$
M$
XNNXNXNNNNXN$
-,PNXNXNNNNXN$
+,P$
2$
7&$
-e&$
-,&$
-7&$
-0&$
+e$
+,$
+7$
+0$
.0$
Cbb$

Diff KO :(
Grade: 0